src/PKSim.Core/Model/Simulation.cs

```diff
@@ -344,6 +345,18 @@ public virtual void RemoveUsedObservedData(DataRepository dataRepository)
          HasChanged = true;
       }
 
+      /// <summary>
+      ///    Remove the output mappings mapped to the dataRepository
+      /// </summary>
+      public void RemoveOutputMappings(DataRepository dataRepository)
+      {
+         var outputsMatchingDeletedObservedData = OutputMappings.OutputMappingsUsingDataRepository(dataRepository).ToList();
```

Comment:
in OutputMappings.cs in Core all the existing functions return IEnumerables ( AllDataRepositoryMappedFor, AllOutputMappingsFor, AllOutputsMappedFor ) since they return the results of LINQ queries. This is the main reason why I have kept this return type.....

Reply:
ok ....